apps/web/src/components/Editor.tsx

```diff
@@ -65,7 +67,14 @@ export const extensions = [
   Highlight,
   Typography,
   IpfsImage.extension,
-  TextAlign,
+  TextAlign.configure({
+    types: ["heading", "paragraph"],
```

Comment:
What other types did it apply to?

Reply:
Not sure, I just moved this configuration. I'll pass this question on to @Kenny-Gin1

Reply:
> What other types did it apply to?

I think it was only them as well. If there’s other scenarios we want them in I can research